### PR TITLE
Event Organization code add

### PR DESCRIPTION
**Pull request by kunal232i**
## Description

- Created Event Organization code in solidity
- Added Readme file to it.
- The functionalities were displayed in accordance with the code.
- completed issue #107 

## Type of change

- [x] Add a new project

## Domain of Contribution

- [x] Project using only Solidity

- [x] Project Test on the Remix IDE

## Checklist ✓

- [x] l have performed a self-review of my own code or work.

- [x] I have commented on my code, particularly in hard-to-understand areas.

images

### CODE REVIEW

File: Level1/Event_Organization/eventOrganization.sol [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        uint maxTickets;
+        uint ticketRemain;
+    }
+        
+    //Mapping is used in this case to store a Multiple number of events.
+    mapping(uint=>Event) public events;
+    //This is Nested Mapping, which is used to allow multiple users to participate in multiple events.
+    mapping(address=>mapping(uint=>uint)) public tickets;
+    uint public nextId;
+
```
Comment (Vikash-8090-Yadav) [inline Level1/Event_Organization/eventOrganization.sol:19]: Explains about this function

File: Level1/Event_Organization/eventOrganization.sol [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    
+    function createEvent(string memory name,uint date,uint price,uint maxTickets) external{
+        // Require is used to determine whether or not the condition is satisfied
+    require(date>block.timestamp,"You can organize event for future date");
+    require(maxTickets>0,"You can organize event only if you create more than 0 tickets");
+    events[nextId] = Event(msg.sender,name,date,price,maxTickets,maxTickets);
+    nextId++;
+    }
+ 
+
```
Comment (Vikash-8090-Yadav) [inline Level1/Event_Organization/eventOrganization.sol:29]: Explains about this function and   their parametre.

**Review by Vikash-8090-Yadav — CHANGES_REQUESTED** [review]

**Review by Vikash-8090-Yadav — APPROVED** [review]

File: Level1/Event_Organization/eventOrganization.sol (hunk 2 above)
Comment (Vikash-8090-Yadav) [inline Level1/Event_Organization/eventOrganization.sol:29]: Explains about this function and parametere

File: Level1/Event_Organization/eventOrganization.sol [hunk 3]
```diff
@@ -0,0 +1,39 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity >=0.5.0 <0.9.0;
+
```
Comment (Vikash-8090-Yadav) [inline Level1/Event_Organization/eventOrganization.sol:3]: Expplain about the event and their parametere ,

File: Level1/Event_Organization/eventOrganization.sol [hunk 4]
```diff
@@ -0,0 +1,39 @@
+//SPDX-License-Identifier: Unlicense
```
Comment (Vikash-8090-Yadav) [inline Level1/Event_Organization/eventOrganization.sol:1]: **All should written in the comments.**

**Comment by kunal232i** [comment]
Hello, @Vikash-8090-Yadav completed the changes, have a look at it now [Event Organization](https://github.com/kunal232i/Solidity-Pathshala/tree/main/Level1/Event_Organization).

**Comment by Vikash-8090-Yadav** [comment]
@kunal232i LGTM ,your codes lookes great and u have u given nice comments . 

  i will merge this afer merging previous merge.This will not effect your Hacktoberfest profile .Please let me know if something other than this happen.

**Pull request merged**